python/fusion_engine_client/messages/configuration.py

```diff
@@ -207,6 +208,13 @@ class NmeaMessageType(IntEnum):
     PQTMVER_SUB = 1204
     PQTMTXT = 1205
 
+class IonoDelayModel(IntEnum):
+    OFF = 0
+    KLOBUCHAR = 1
```

Comment:
Maybe we should add `DEFAULT` to this list. That way we get behaviors like "Off", "On", or "something specific". Maybe I am just saying that to register the thought. In the interest of getting a point release out today, let's not make that change now.

Reply:
We should probably not add default to config options to specific values. We have a planned implementation for "not set" and there's already a way to set a config option to default generally.

Reply:
So are we planning to _not_ add a default option here then? The way I'll set it to work, if the user doesn't set it at all, then it'll default to using the models we currently use.

Reply:
We should not at a default option here. @wbrannon I'm not sure what you mean by "if the user doesn't set it at all". Typically we handle defaults in the default user config json files, so I assume that's what you're talking about?

Reply:
@axlan yes the way you described it is what I mean. Thanks for clarifying.